code/game/gamemodes/cult/cult_items.dm

```diff
@@ -116,19 +117,224 @@
 	armor = list(melee = 50, bullet = 30, laser = 50,energy = 20, bomb = 25, bio = 10, rad = 0)
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 
-
 /obj/item/clothing/head/helmet/space/cult
-	name = "nar-sian bruiser's helmet"
-	desc = "A heavily-armored helmet worn by warriors of the Nar-Sian cult. It can withstand hard vacuum."
+	name = "nar-sien hardened helmet"
+	desc = "A heavily-armored helmet worn by warriors of the Nar-Sien cult. It can withstand hard vacuum."
 	icon_state = "cult_helmet"
 	item_state = "cult_helmet"
 	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
 
 /obj/item/clothing/suit/space/cult
-	name = "nar-sian bruiser's armor"
+	name = "nar-sien hardened armor"
 	icon_state = "cult_armor"
 	item_state = "cult_armor"
-	desc = "A heavily-armored exosuit worn by warriors of the Nar-Sian cult. It can withstand hard vacuum."
-	w_class = 3
+	desc = "A heavily-armored exosuit worn by warriors of the Nar-Sien cult. It can withstand hard vacuum."
+	w_class = 2
 	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade,/obj/item/weapon/tank/internals/)
 	armor = list(melee = 70, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
+
+/obj/item/clothing/suit/space/cult/pickup(mob/living/user)
+	..()
+	if(!iscultist(user))
+		user << "<span class='cultlarge'>\"I wouldn't advise that.\"</span>"
+		user << "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>"
+		user.unEquip(src, 1)
+		user.Dizzy(30)
+		user.Weaken(5)
+		
+
+/obj/item/weapon/sharpener/cult
+	name = "eldritch whetstone"
+	desc = "A block, empowered by dark magic. Sharp weapons will be enhanced when used on the stone."
+	used = 0
+	increment = 5
+	max = 40
+	prefix = "darkened"
+
+/obj/item/clothing/suit/hooded/cultrobes/cult_shield
+	name = "empowered cultist armor"
+	desc = "Empowered garb which creates a powerful shield around the user."
+	icon_state = "cult_armor"
+	item_state = "cult_armor"
+	armor = list(melee = 50, bullet = 40, laser = 50,energy = 30, bomb = 50, bio = 30, rad = 30)
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade)
+	var/current_charges = 3
```

Comment:
This might end up disgustingly unfun as soon as the cultist in question gets their hands on a gun, and an officer has to shoot them 4 times (vs the cultist shooting them once) to down them, but I guess cultists always lose anyway so whatever.